tests/test_models.py

```diff
@@ -404,6 +432,90 @@ def test_new_clarification_questions_get_added_to_the_end(self):
             assert brief.clarification_questions[0].question == "How?"
             assert brief.clarification_questions[1].question == "When"
 
+    def test_awarded_brief_response_when_there_is_an_award(self):
+        with self.app.app_context():
+            self.setup_dummy_suppliers(1)
+            brief = Brief(data={}, framework=self.framework, lot=self.lot, published_at=datetime(2000, 1, 1))
+            brief_response1 = BriefResponse(brief=brief, supplier_id=0, submitted_at=datetime.utcnow(), data={})
+            brief_response2 = BriefResponse(brief=brief, supplier_id=0, submitted_at=datetime.utcnow(), data={})
+            brief_response2.award_details = {"confirmed": "details"}
+            brief_response2.awarded_at = datetime(2016, 12, 12, 1, 1, 1)
+            db.session.add_all([brief, brief_response1, brief_response2])
+            db.session.commit()
+
+            brief = Brief.query.get(brief.id)
+            assert brief.awarded_brief_response.id == brief_response2.id
+
+    def test_no_awarded_brief_response_if_brief_responses_but_no_award(self):
+        with self.app.app_context():
+            self.setup_dummy_suppliers(1)
+            brief = Brief(data={}, framework=self.framework, lot=self.lot, published_at=datetime(2000, 1, 1))
+            brief_response1 = BriefResponse(brief=brief, supplier_id=0, submitted_at=datetime.utcnow(), data={})
+            brief_response2 = BriefResponse(brief=brief, supplier_id=0, submitted_at=datetime.utcnow(), data={})
+            db.session.add_all([brief, brief_response1, brief_response2])
+            db.session.commit()
+
+            brief = Brief.query.get(brief.id)
+            assert brief.awarded_brief_response is None
+
+    def test_no_awarded_brief_response_if_no_brief_responses(self):
+        with self.app.app_context():
+            brief = Brief(data={}, framework=self.framework, lot=self.lot, published_at=datetime(2000, 1, 1))
+            db.session.add(brief)
+            db.session.commit()
+
+            brief = Brief.query.get(brief.id)
+            assert brief.awarded_brief_response is None
+
+    def _setup_brief_and_awarded_brief_response(self, context, awarded_at=True, pending=True):
```

Comment:
Whereas this test class is HUGE... Perhaps I'll split out the awards-related tests.